src/Tools/dotnet-user-jwts/src/Helpers/Jwt.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Authentication.JwtBearer.Tools;
+
+public record Jwt(string Id, string Name, string Audience, DateTimeOffset NotBefore, DateTimeOffset Expires, DateTimeOffset Issued, string Token)
```

Comment:
nit: don't know if style for records is different, but aren't parameter names supposed to be lower? just reads funny to me like this

Reply:
They're actually properties in this syntax, so PascalCase is the norm.